console/src/view/tasks.rs

```diff
@@ -10,8 +11,10 @@ use tui::{
     text::{self, Span, Spans},
     widgets::{Cell, Paragraph, Row, Table, TableState},
 };
-#[derive(Clone, Debug, Default)]
+
+#[derive(Debug)]
 pub(crate) struct List {
+    warnings: Vec<Box<dyn warnings::Warning<Task>>>,
```

Comment:
Ah, after reading through the file, I realized I misunderstood this field. I thought this was all the warnings that had been found in the tasks, but it's the list of warnings to check for. Maybe a field name of `lints`, or a doc comment?